Documentation/doc/Documentation/Developer_manual/create_and_use_a_cmakelist.txt

```diff
@@ -14,7 +14,7 @@ target_link_libraries(my_executable CGAL::CGAL)
 Other \cgal libraries are linked similarly. For example, with `CGAL_Core`:
 
 \code
-find_package(CGAL REQUIRED COMPONENTS Core)
+find_package(CGAL COMPONENTS Core)
```

Comment:
why that change?

Reply:
It's documentation on using CGAL in an external program so it is that external program's responsability to use whatever flag they want. And the other `find_package(CGAL)` just a couple lines above did that too by not having `REQUIRED`.

Reply:
I agree with Mael. See https://doc.cgal.org/5.5/Manual/devman_create_and_use_a_cmakelist.html:
  - either we remove `REQUIRED` in the second occurrence,
  - or we add it as well in the first occurrence.

The current situation can confuse readers and lead them to think that `REQUIRED` and `COMPONENTS` are linked.

Reply:
Actually, I would add `REQUIRED` everywhere in that documentation page. There is a third occurrence a bit further in the page, and it has `REQUIRED`.